### PR TITLE
[SDK-7095] Added missing symbologies to the documentation.

### PR DESCRIPTION
**Pull request by moritzhartmeier**

### CODE REVIEW

File: docs/Scandit.java [hunk 1]
```diff
@@ -220,6 +224,10 @@
              * \brief PDF417 barcode symbology.
              */
             PDF417,
+            /**
+             * \brief PDF417 barcode symbology.
```
Comment (TomekJurkowski) [inline docs/Scandit.java]: Shouldn't it say "Micro PDF417" ?

File: docs/Scandit.java [hunk 2]
```diff
@@ -271,7 +275,15 @@
              * as well: \ref SYMBOLOGY_EAN13, \ref SYMBOLOGY_UPCA, \ref SYMBOLOGY_UPCE, or
              * \ref SYMBOLOGY_EAN8 and the maximum number of codes per frame has to be set to at least 2.
              */
-            TWO_DIGIT_ADD_ON
+            TWO_DIGIT_ADD_ON,
+            /**
+             * \brief Royal Dutch TPG Post KIX
```
Comment (TomekJurkowski) [inline docs/Scandit.java]: Dots are missing at the end of the comments for KIX and RM4SCC :P

File: docs/Scandit.java [hunk 3]
```diff
@@ -244,10 +252,6 @@
              * \brief Aztec 2D barcode symbology.
              */
             AZTEC,
-            /**
-             * @brief MicroQR 2D barcode symbology.
-             */
-            MICRO_QR,
             /**
              * @brief DotCode 2D barcode symbology.
```
Comment (TomekJurkowski) [inline docs/Scandit.java]: bonus question: is @brief an equivalent of \brief ?

Comment (moritzhartmeier) [reply docs/Scandit.java]: yep it is, doxygen will handle both the same way, i changed this to backslash though for consistency.

**Pull request merged**